Inter-views/Inter-views
# Modal

## **Types of Changes**


## **Purpose**
- manage state with redux pattern

## **Approach**
- utilize Redux Tool Kit to generate slices that manage actions, dispatch, and reducers

## **How to Test**
- npm run dev

click 'add card' button on dashboard // should render modal with console log in browser of changed state
